setup.py

```diff
@@ -77,6 +87,10 @@ def run(self):
         "Topic :: Software Development :: Compilers",
         "Topic :: Software Development :: Libraries"
         ],
+    setup_requires=["Opcodes==0.3.10", "six"],
+    install_requires=["Opcodes==0.3.10", "six", "enum34"],
```

Comment:
`opcodes` is redundant here. It is needed to generate instruction in PeachPy, but after PeachPy is installed, `opcodes` is no longer needed